tests/integration_tests/modules/test_combined.py

```diff
@@ -150,17 +149,16 @@ def test_deprecated_message(self, class_client: IntegrationInstance):
         # user-data. Pass 22.2 in against the client's version_boundary.
         if lifecycle.should_log_deprecation("22.2", version_boundary):
             deprecated_messages.append(boundary_message)
-            verify_clean_boot(
-                class_client, require_deprecations=deprecated_messages
-            )
         else:
             # Expect the distros deprecated call to be redacted.
             # jsonschema still emits deprecation log due to changed_version
             # instead of deprecated_version
-            verify_clean_boot(
-                class_client, require_deprecations=deprecated_messages
-            )
             assert f"[INFO]: {boundary_message}" in log
+        verify_clean_boot(
```

Comment:
Nothing that needs to change here, but it does seem a little odd calling `verify_clean_boot` multiple times in multiple tests in the class given that they're all operating on the same instance and the log won't be any different between tests.

Reply:
Yeah I had wondered about just consolidating the deprecation test and the no_problems test.  I have though consolidated the rsyslog verify_clean_boot call with testy_no_problems case as they are both testing and asserting the same log warning condition based on presence of rsyslogd.